src/z3c/recipe/sphinxdoc/__init__.py

```diff
@@ -168,9 +171,11 @@ def install(self):
     update = install
 
 
-def main(projects):
+def main(projects, argv=None, exit_on_error=False):
```

Comment:
There was actually a brief period of time where `sphinx.main` *did* exit. That introduced a bug in this package (#2). But later sphinx removed it again.